### PR TITLE
OSDD : Open Source Driven Development

### PR DESCRIPTION
**Pull request by FanJups**

### CODE REVIEW

File: data/authors/fanjups.md
Comment (RMPR) [inline data/authors/fanjups.md]:
```suggestion
Some contributions:
```

**Review by RMPR — COMMENTED** [review]
Great work, can you squash the two last commits into one? It does not seem clear to me why they are separate. Also, a more generic comment, can you convert the images to optimized JPeG? This is to be more space efficient and avoid people from Cameroon cloning with limited internet to be stuck.

File: data/authors/fanjups.md
Comment (FanJups) [reply data/authors/fanjups.md]: Ok thanks,  will apply

**Comment by tericcabrel** [comment]
Hello @FanJups 👋🏾
We fixed the CI error; please can you update your repository with the latest changes and update this pull request? 🙏🏾

**Comment by FanJups** [comment]
> Great work, can you squash the two last commits into one? It does not seem clear to me why they are separate. Also, a more generic comment, can you convert the images to optimized JPeG? This is to be more space efficient and avoid people from Cameroon cloning with limited internet to be stuck.

Sure, thanks for the tips

**Comment by FanJups** [comment]
These articles helped  me to squash https://www.freecodecamp.org/news/git-squash-commits/ https://stackoverflow.com/questions/5667884/how-to-squash-commits-in-git-after-they-have-been-pushed

**Comment by FanJups** [comment]
> Hello @FanJups 👋🏾 We fixed the CI error; please can you update your repository with the latest changes and update this pull request? 🙏🏾

done

**Comment by FanJups** [comment]
@RMPR @tericcabrel , please check again 💯

File: data/blog/open-source-driven-development.mdx
Comment (RMPR) [inline data/blog/open-source-driven-development.mdx]:
```suggestion
> And while a lot of big company FOSS usage feeds back to the community, a lot do not.
```

File: data/blog/open-source-driven-development.mdx
Comment (RMPR) [inline data/blog/open-source-driven-development.mdx:113]: I think it would be better if this was a separate post called "My open source journey" for example, what do you think?

Comment (FanJups) [reply data/blog/open-source-driven-development.mdx:113]: Makes sense but given the plan, it's better this way.

Comment (FanJups) [reply data/blog/open-source-driven-development.mdx:113]: I thought about the same but my April 2 
 has the same structure : https://speakerdeck.com/fanjups/open-source-driven-development

Comment (RMPR) [reply data/blog/open-source-driven-development.mdx:113]: > has the same structure : https://speakerdeck.com/fanjups/open-source-driven-development

The fact that it is a presentation allows for flexibility in the format. However, this is an article, and judging by the summary:

> It’s about how Open Source can help individual or organization to get better and better

I think this is slightly off-topic. I do recognize that it is more aesthetics than actual problems with your work. As such, don't consider me a blocker for this PR.

Comment (FanJups) [reply data/blog/open-source-driven-development.mdx:113]: Yes, I understand,  no worries. Thanks to you, I learned how to squash commits using git commands 😊. I used to do that  by clicking on github button before merging PR. 

It's just that given the structure,  I mean the whole idea, readers should  not go to another url,  It's better to stay on the same article.

Comment (FanJups) [reply data/blog/open-source-driven-development.mdx:113]: Also, this part is the core of the article because we can clearly see how I have improved,  hence " ... get better and better ".  I also added the organisation part to illustrate.

Comment (FanJups) [reply data/blog/open-source-driven-development.mdx:113]: Consequently,  it's better to keep the core  with other parts.

Comment (FanJups) [reply data/blog/open-source-driven-development.mdx:113]: Thanks for your review 😊

Comment (RMPR) [reply data/blog/open-source-driven-development.mdx:113]: When you make the CI green we can merge.

**Comment by FanJups** [comment]
@tericcabrel I got this CI error

```
#21 ERROR: denied: installation not allowed to Write organization package
------
 > pushing ghcr.io/osscameroon/blog:ec7382128144d1c30568651e6a7667662b814aa8 with docker:
------
ERROR: denied: installation not allowed to Write organization package
Error: buildx failed with: ERROR: denied: installation not allowed to Write organization package
```

**Comment by RMPR** [comment]
This has been fixed https://github.com/osscameroon/blog/pull/24
Can you rebase your branch on the latest master?

**Comment by RMPR** [comment]
> @tericcabrel I got this CI error
> 
> ```
> #21 ERROR: denied: installation not allowed to Write organization package
> ------
>  > pushing ghcr.io/osscameroon/blog:ec7382128144d1c30568651e6a7667662b814aa8 with docker:
> ------
> ERROR: denied: installation not allowed to Write organization package
> Error: buildx failed with: ERROR: denied: installation not allowed to Write organization package
> ```

This has been fixed https://github.com/osscameroon/blog/pull/24
Can you rebase your branch on the latest master?

**Comment by FanJups** [comment]
> > @tericcabrel I got this CI error
> > ```
> > #21 ERROR: denied: installation not allowed to Write organization package
> > ------
> >  > pushing ghcr.io/osscameroon/blog:ec7382128144d1c30568651e6a7667662b814aa8 with docker:
> > ------
> > ERROR: denied: installation not allowed to Write organization package
> > Error: buildx failed with: ERROR: denied: installation not allowed to Write organization package
> > ```
> 
> This has been fixed #24 Can you rebase your branch on the latest master?

Looks like, its already done days ago
Screenshot_20240415_151330_Chrome

**Comment by FanJups** [comment]
> > > @tericcabrel I got this CI error
> > > ```
> > > #21 ERROR: denied: installation not allowed to Write organization package
> > > ------
> > >  > pushing ghcr.io/osscameroon/blog:ec7382128144d1c30568651e6a7667662b814aa8 with docker:
> > > ------
> > > ERROR: denied: installation not allowed to Write organization package
> > > Error: buildx failed with: ERROR: denied: installation not allowed to Write organization package
> > > ```
> > 
> > 
> > This has been fixed #24 Can you rebase your branch on the latest master?
> 
> Looks like, its already done days ago Screenshot_20240415_151330_Chrome

Screenshot_20240415_151911_Chrome

**Comment by FanJups** [comment]
I also tried again to rebase but everything is up to date

```ps
PS C:\GitHub\blog> git status
On branch open-source-driven-development
Your branch is up to date with 'origin/open-source-driven-development'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        package-lock.json

nothing added to commit but untracked files present (use "git add" to track)
PS C:\GitHub\blog> git rebase upstream/main
Current branch open-source-driven-development is up to date.
PS C:\GitHub\blog>
```

**Comment by tericcabrel** [comment]
Hello @FanJups I'm investigating 👍🏾

**Comment by FanJups** [comment]
Ok thanks @tericcabrel , please any updates ?

**Comment by tericcabrel** [comment]
After checking with @FanJups, it seems like there is a permission issue with the GitHub container registry when the PR comes from a fork. I'm still investigating!

**Comment by FanJups** [comment]
@tericcabrel , looks like everything is ok now after @elhmn 's commit

**Comment by FanJups** [comment]
@tericcabrel , I can not invite you as a reviewer given that I am not admin on this repo

**Comment by FanJups** [comment]
@elhmn just changed the number of reviewers from 2 to 1 then with @RMPR pending approval, we are all set.

**Review by tericcabrel — APPROVED** [review]

**Review by RMPR — APPROVED** [review]

**Pull request merged**
